prow/config/config.go

```diff
@@ -535,6 +542,26 @@ type Plank struct {
 	JobURLPrefixDisableAppendStorageProvider bool `json:"jobURLPrefixDisableAppendStorageProvider,omitempty"`
 }
 
+type ProwJobDefaultEntry struct {
+	// Matching/filtering fields. All filters must match for an entry to match.
+
+	// OrgRepo matches against the "org" or "org/repo" that the presubmit or postsubmit
+	// is associated with. If the job is a periodic, extra_refs[0] is used. If the
+	// job is a periodic without extra_refs, the empty string will be used.
+	// If this field is omitted all jobs will match.
+	OrgRepo string `json:"repo,omitempty"`
```

Comment:
Nit: maybe `OrgOrRepo` and also make the json tag reflect that name?

Reply:
Keeping OrgRepo as the behavior is the same as defaultDecorationConfig and I want the naming to be consistent